### PR TITLE
failing FusionReduction4

### PR DESCRIPTION
**Pull request by jjsjann123**
removed runTestKernel.

### CODE REVIEW

**Comment by jjsjann123** [comment]
I'm still failing a test `FusionReduction4()`.

ExprEvaluator is failing an assert. I've seen it before after the computeAt refactor that a bug in my scheduling caused that. I'll take a look at the test later, but this should be good for review now.

**Review by csarofeen — APPROVED** [review]

File: test/cpp/jit/test_gpu.cpp [hunk 1]
```diff
@@ -1204,8 +1236,16 @@ void testGPU_FusionCodeGen2() {
   tv3->axis(-1)->parallelize(ParallelType::TIDx);
 
   prog.setDevice(0);
-  prog.grid(4);
-  prog.block(8);
+  setupLaunchConfig(
+      prog.fusion(),
+      8, // tid_x
```
Comment (naoyam) [inline test/cpp/jit/test_gpu.cpp:1241]: This should be 4?

File: test/cpp/jit/test_gpu.cpp [hunk 2]
```diff
@@ -1204,8 +1236,16 @@ void testGPU_FusionCodeGen2() {
...
+      1, // tid_y
+      1, // tid_z
+      4, // gid_x
```
Comment (naoyam) [inline test/cpp/jit/test_gpu.cpp:1244]: 8?

File: test/cpp/jit/test_gpu.cpp (hunk 1 above)
Comment (naoyam) [reply test/cpp/jit/test_gpu.cpp:1241]: Sorry, no. Please ignore.

**Review by naoyam — APPROVED** [review]
LGTM

**Review by kevinstephano — COMMENTED** [review]
Looks good to me!

**Comment by jjsjann123** [comment]
Tests are passing locally. Merging after CI.

**Pull request merged**